src/dmqproto/client/legacy/internal/request/PushMultiRequest.d

```diff
@@ -122,7 +123,7 @@ public scope class PushMultiRequest : IMultiChannelRequest
                     (__FILE__, __LINE__));
                 break;
             case MultipleNodeQuery:
-                assert(false, "pushMulti: request is not multiple node");
+                verify(false, "pushMulti: request is not multiple node");
```

Comment:
Just out of interest: why no `assert(false);` here? (In contrast with https://github.com/sociomantic-tsunami/dmqproto/pull/41/commits/6df1988dd3e174f4eb1c3be109389abc6f456b07#diff-a7ea079493f61560a12b211fd755f81fR109.)

Reply:
There should be `assert(false)`. I oversaw it because DMD doesn't complain -- I don't understand why.

Reply:
In D2 there would be no case fall-through. Perhaps dmd1 doesn't complain because the next case i.e. `default` starts with `assert(false)`?